### ISSUE TITLE
Error on http://purl.org/dc/terms/hasVersion: The named property should be specified for this resource.

### ISSUE THREAD

**Issue by berezovskyi** [oslc-op/oslc-specs#384]
From https://app.circleci.com/pipelines/github/oslc-op/oslc-specs/509/workflows/78185a01-c2fd-4eff-8453-32aec6d67ae0/jobs/619

**Comment by berezovskyi** [comment]
@ndjc how should this be fixed/

**Comment by ndjc** [comment]
Add the dcterms:hasVersion property to the owl:ontology and oslcResourceShapeConstraintsresources.
I copied the list of required new publishing properties from one of the specs, forget which:

Mandatory if -C is used:
```
dcterms:title
dcterms:license
dcterms:hasVersion
dcterms:source
dcterms:isPartOf
```

Optional even with -C:
```
dcterms:description
dcterms:modified
dcterms:publisher
dcterms:issued
dcterms:dateCopyrighted
rdfs:label
rdfs:seeAlso, zero or many
```

`rdfs:label` is mandatory for a vocabulary, and optional (zero or one) for a `ResourceShapeConstraints`.

Note that ShapeChecker does not have a way to categorize required properties by document status, so we cannot currently require it to check (for example) that `dcterms:issued` is set if and only if the document status is one of PS, COS, OS, etc.

**Comment by berezovskyi** [comment]
Ah yes, I missed the fact the hasVersion was also added as a requirement for the vocab. Fixed in #383
